docs/packaging/advanced-config/local-repository.md

```diff
@@ -21,63 +21,73 @@ sudo eopkg install solbuild-config-local-unstable
+To copy all `.eopkg` files within a directory to the local repository, use the following command:
 
-## Best practices when working with a solbuild local repository
+```bash
+cp *.eopkg /var/lib/solbuild/local
+```
 
-There are some important things to know when working with local repositories, as they may lead to issues later on.
+### Using the local repository to build a package
 
-- Solbuild will use your version of a package from the local repo regardless of whether there's a higher release in the Solus repository. Therefore you should only use `go-task build-local` when required and also remove old packages from the local repo when no longer needed.
-- If the package is already installed in the solbuild image, the release must be higher for it to be installed.
+With the `.eopkg` files now present in the local repository, you can use them to build a package by running `go-task build-local`, rather than just `go-task`, and `solbuild` will prefer your packages over packages found in the Solus repository.
 
-## Eopkg and local repositories
+Every time you run `go-task build-local`, all `eopkg` files in the local repository will be re-indexed.
 
-As alluded to above, eopkg supports resolving packages from more than one repository.
+### Best practices when working with a solbuild local repository
 
-In practice, this can be leveraged to make the developer workflow more convenient by ensuring that eopkg pulls from the local solbuild repository.
+There are some important things to know when working with local repositories, as they may lead to issues later on.
 
-### Generating an eopkg index for the local solbuild repository
+- `solbuild` will use your version of a package from the local repository regardless of whether there's a higher release in the Solus repository. Therefore you should only use `go-task build-local` when required and also remove old packages from the local repository when they are no longer needed.
+- If the package is already installed in the `solbuild` image, the release must be higher for it to be installed.
 
-Before packages from the local solbuild repository can be resolved, an eopkg index file needs to be created.
+## Installing packages from the local repository index
 
-As mentioned earlier, the local solbuild repo installed by the `solbuild-config-local-unstable` package lives in `/var/lib/solbuild/local`.
+`eopkg` supports resolving packages from more than one repository. This can be used to configure `eopkg` to prefer installing local repository packages over packages from the Solus repository.
 
-To generate or refresh the eopkg index in `/var/lib/solbuild/local`, simply run:
+This requires an existing local repository index. If you have used the command `go-task build-local`, then an index was created automatically. Otherwise, you must create one
```

Comment:
```suggestion
This requires an existing local repository index. If you have used the command `go-task build-local`, then an index was created automatically. Otherwise, you must create one.
```